include/drivers/smmu/smmuv2.h

```diff
@@ -8,19 +8,23 @@
 
 #include <plat/machine/devices_gen.h>
 
+#if defined(CONFIG_PLAT_ZYNQMP)
```

Comment:
We could potentially extract the SMMU address from DTS, just as we did for GIC.

Reply:
It already is and should be generated as the constant `SMMU_PPTR`

Reply:
Actually the mapping automatically extracted from the DTS for the SMMU is only the first 4k.